src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -2041,8 +2041,11 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 /* - General Card Texts */
 
 "Statistics_Card_From_Nationwide" = "Bundesweit";
+
 "Statistics_Card_From_CWA" = "über die Corona-Warn-App";
 
+"Statistics_Card_LinkCard_buttonTitle" = "Externen Link öffnen  ";
+
```

Comment:
is this extra space intentional ?

Reply:
This is the space between the button title attributed text and the text attachment (the sf symbol)

Reply:
@flxschmidt Please add the space with code then as the translation delivery doesn't necessarily take care of spaces.